### PR TITLE
Review.rst: Add more details for reviewers

### PR DESCRIPTION
**Pull request by jayvdb**
Split off from https://github.com/coala/coala/pull/3657

### CODE REVIEW

File: docs/Developers/Review.rst [hunk 1]
```diff
@@ -105,6 +105,10 @@ For the Reviewers
 
 -  All the pull requests waiting to be reviewed can be found at :
    https://coala.io/review.
+-  Check the commit message.
+-  Read and try to understand the code. If something looks ineffective or
+   bug prone, leave a comment.
+   If in doubt, let the code-writer explain his though process to understand it.
```
Comment (jayvdb) [inline docs/Developers/Review.rst]: gender.

though -> thought

Comment (jayvdb) [reply docs/Developers/Review.rst]: Better yet, though process -> reasoning.

**Comment by sims1253** [comment]
lgtm 
ack 2d44c08b49f0a5d2744445536734772fcea3c08d

File: docs/Developers/Review.rst [hunk 2]
```diff
@@ -105,6 +105,11 @@ For the Reviewers
 
 -  All the pull requests waiting to be reviewed can be found at :
    https://coala.io/review.
+-  Check the commit message.
```
Comment (prnvdixit) [inline docs/Developers/Review.rst]: Maybe add "Check that the changes haven't been pushed from master branch" to/after this ^^ point - Else LGTM as well. 😄

Comment (jayvdb) [reply docs/Developers/Review.rst]: Then we have reviewers telling newcomers to close their PRs, even if the PR is good and can be merged quickly.

You've raised an important issue, which should be in this guide, but it needs to be a separate addition with lots of careful wording

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**